src/browser/components/events/event_list.jsx

```diff
@@ -34,11 +34,9 @@ EventList = React.createClass({
         }, this);
 
         return (
-            <div className='pure-u-1 pure-u-lg-1-3'>
-                <ul className='event-list'>
-                    {eventComponents}
-                </ul>
-            </div>
+            <ul className='event-list'>
```

Comment:
This div was unnecessary. You can style the ul, especially since the component is just a list.